lib/google_contacts_api/contacts_accessor.rb

```diff
@@ -25,5 +25,37 @@ def get_contacts(params = {})
       end
       GoogleContactsApi::ContactSet.new(response.body, @api)
     end
+
+    def post_contact(contact)
+      params = {}
+      headers = {
+          'Content-type' => 'application/json'
+          }
+      body = contact.json
+      params[:body] = body
+      url = "contacts/default/full"
```

Comment:
This is a "magic constant" (which is not ideal): https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants
Also, at the moment we are at a Connacence of Value, where a constant would give us Connascence of Name https://en.wikipedia.org/wiki/Connascence_(computer_programming)#Connascence_of_Name_.28CoN.29

In short, this should be a class constant.